core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```diff
@@ -119,11 +119,10 @@
         </div>
 
         <ul class="dropdown-menu" aria-labelledby="discardButtonPopup">
-<!-- The 'disabled' class is added when autosave is in progress. This disables user interactions with the discard button, preventing changes from being discarded before they are saved. -->
-          <div class="e2e-test-mobile-exploration-discard-tab" (click)="discardChanges()" [class.disabled]="autosaveIsInProgress">
+          <button class="btn btn-light e2e-test-mobile-exploration-discard-tab" (click)="discardChanges()" [disabled]="autosaveIsInProgress">
```

Comment:
Is it necessary to change the HTML tag from div to button? We try not to modify the HTML elements.

Reply:
Just a note -- while the general idea of making minimal changes is correct, "button" is actually more semantic here, so I think this change is OK (we should label buttons as buttons for accessibility).